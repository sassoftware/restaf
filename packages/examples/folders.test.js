let testFunctions = require("./testFunctions");
let testInfo;
beforeAll(async () => {
	testInfo = await require('./lib/setupAll')();
});

test('folders - paginate thru folders', async () => {
	expect.assertions();
	let l = await testFunctions.foldersPaginate(testInfo);
	expect(l).toMatchSnapshot();
});