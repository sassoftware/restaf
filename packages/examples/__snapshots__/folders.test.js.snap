// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`folders - paginate thru folders 1`] = `"done"`;
